src/it/MPMD-268-deprecated-rules/verify.groovy

```diff
@@ -19,4 +19,6 @@
 
 File buildLog = new File( basedir, 'build.log' )
 assert buildLog.exists()
-assert buildLog.text.contains( "[WARNING] Use Rule name category/java/codestyle.xml/ControlStatementBraces instead of the deprecated" )
+// Some versions output `WARNING:` and others `[WARNING]`
```

Comment:
Where do you see this difference?
Running m-pmd-p with PMD 6.13.0 on several platforms doesn't show this problem: https://builds.apache.org/view/M-R/view/Maven/job/maven-box/job/maven-pmd-plugin/job/MPMD-275-PMD-6.13.0/

Reply:
I have ran the tests twice using JDK11 and JDK12 on macOS 10.14.1. I don't know which output what now since I am afk.

Thanks for the response. When do you expect the release of 6.13.0 change to maven?

Cheers!

Reply:
I want to fix a couple of other issues first, so it'll take a few days...

Reply:
can't reproduce the above issue again. Closing

Reply:
btw - if you don't want to wait for a new release, you can also just update PMD yourself: https://maven.apache.org/plugins/maven-pmd-plugin/examples/upgrading-PMD-at-runtime.html

Reply:
Thanks for the tip!